presto-parquet/src/test/java/io/prestosql/parquet/reader/TestMetadataReader.java

```diff
@@ -130,95 +130,97 @@ public void testReadStatsBinary(Optional<String> fileCreatedBy)
+        testReadStatsBinaryUtf8OldWriter(NO_CREATED_BY, "é", "a", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR, "é", "a", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR_1_8, "é", "a", null, null);
 
         // [é, ê]: no common prefix, first different are both not ASCII
-        testReadStatsBinaryUtf8OldWriter(NO_CREATED_BY, 13, "é".getBytes(UTF_8), "ê".getBytes(UTF_8), 13, null, null);
-        testReadStatsBinaryUtf8OldWriter(PARQUET_MR, 13, "é".getBytes(UTF_8), "ê".getBytes(UTF_8), 13, null, null);
-        testReadStatsBinaryUtf8OldWriter(PARQUET_MR_1_8, 13, "é".getBytes(UTF_8), "ê".getBytes(UTF_8), 13, null, null);
+        testReadStatsBinaryUtf8OldWriter(NO_CREATED_BY, "é", "ê", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR, "é", "ê", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR_1_8, "é", "ê", null, null);
 
         // [aé, aé]: min = max (common prefix, first different are both not ASCII)
-        testReadStatsBinaryUtf8OldWriter(NO_CREATED_BY, 13, "aé".getBytes(UTF_8), "aé".getBytes(UTF_8), 13, null, null);
-        testReadStatsBinaryUtf8OldWriter(PARQUET_MR, 13, "aé".getBytes(UTF_8), "aé".getBytes(UTF_8), 13, null, null);
-        testReadStatsBinaryUtf8OldWriter(PARQUET_MR_1_8, 13, "aé".getBytes(UTF_8), "aé".getBytes(UTF_8), 13, "aé".getBytes(UTF_8), "aé".getBytes(UTF_8));
+        testReadStatsBinaryUtf8OldWriter(NO_CREATED_BY, "aé", "aé", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR, "aé", "aé", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR_1_8, "aé", "aé", "aé", "aé");
 
         // [aé, bé]: no common prefix, first different are both ASCII
-        testReadStatsBinaryUtf8OldWriter(NO_CREATED_BY, 13, "aé".getBytes(UTF_8), "bé".getBytes(UTF_8), 13, null, null);
-        testReadStatsBinaryUtf8OldWriter(PARQUET_MR, 13, "aé".getBytes(UTF_8), "bé".getBytes(UTF_8), 13, null, null);
-        testReadStatsBinaryUtf8OldWriter(PARQUET_MR_1_8, 13, "aé".getBytes(UTF_8), "bé".getBytes(UTF_8), 13, "a".getBytes(UTF_8), "c".getBytes(UTF_8));
+        testReadStatsBinaryUtf8OldWriter(NO_CREATED_BY, "aé", "bé", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR, "aé", "bé", null, null);
+        testReadStatsBinaryUtf8OldWriter(PARQUET_MR_1_8, "aé", "bé", "a", "c");
     }
 
-    private void testReadStatsBinaryUtf8OldWriter(Optional<String> fileCreatedBy, int nullCount, byte[] min, byte[] max, int expectedNullCount, byte[] expectedMin, byte[] expectedMax)
+    private void testReadStatsBinaryUtf8OldWriter(Optional<String> fileCreatedBy, String min, String max, String expectedMin, String expectedMax)
     {
         Statistics statistics = new Statistics();
-        statistics.setNull_count(nullCount);
-        statistics.setMin(min);
-        statistics.setMax(max);
+        statistics.setNull_count(13);
```

Comment:
Maybe put the `13` into a constant so that it's clear where it comes from, especially since it's used further down.

Reply:
i will

Reply:
hm... i will not do it. It's like this in every other test method in this class.